bash_aliases.sh

```diff
@@ -1,26 +1,36 @@
+alias seder='sed  's/bin\/activate$//' '            # Cannot remember
 
 #alias python=python2.7 
 
 # Conda Aliases                                                                  
-alias conda-remove-env='conda env remove --name $1'                             
-alias conda-create-env='conda create --name $1'                                 
-alias conda-lsp='conda list --name $1' # List conda packages                                          
-alias conda-ls='conda env list'        # List conda environments                                                  
-                                                                                 
-#Git Aliases                                                                     
-alias git-graph='git log --graph --oneline'                      # A simpler heuristic                                   
-alias git-show-lastn=' bash /home/meow/scripts/gitshowcom.sh $1' # Look behind at the nth commit back in the log.
+alias conda-create-env='conda env create --name'    # Creates a conda enviroment with the name as first argument($1)
+alias conda-remove-env='conda env remove --name'    # Removes a conda enviroment with the name as first argument($1)
+alias conda-eu="conda env update"                   # Updates the current conda environment
+alias conda-ls='conda env list'                     # List conda environments
+alias conda-lsp='conda list --name'                 # List conda packages                                          
+alias conda-i="conda install"                       # Installs a list of packages into a specified conda environment provided as arguments
+alias conda-r="conda remove"                        # Removes a list of packages from a specified conda environment provided as arguments
+
+#Git Aliases                                                
+
+alias git-graph="git log --graph --abbrev-commit --decorate --format=format:'%C(bold green)%h%C(reset) - %C(bold cyan)%aD%C(reset) %C(bold yellow)(%ar)%C(reset)%C(auto)%d%C(reset)%n''%C(white)%s%C(reset) %C(dim white)- %an%C(reset)' --all" # Simple and Beautiful way to display git log
+alias git-show-lastn=' bash $HOME/scripts/gitshowcom.sh' # Look behind at the nth commit back in the log.
 
 # Utils
-function lsize() { du -s ${1}/* | sort -rn | cut -f2- | xargs -d "\n" du -sh }
-alias ls-size='lsize $1wq'
+lsize() { 
+        du -s "${1}"/* 2> >(grep -v '^du: cannot \(access\|read\)' >&2) | sort -rn | \
+        cut -f2- | xargs -d "\n" du -sh 2> >(grep -v '^du: cannot \(access\|read\)' >&2);
+}
+
 alias ls-hidden='ls -d .* --color=auto'  # List hidden files
 
-alias speedread='~/$USER/Downloads/speedread/speedread $1'
+alias speedread='$HOME/Downloads/speedread/speedread'
 
-function bat-help(){ $1 --help | batcat }  #Bat is requird
-alias bathelp='bat-help $1'
+#Bat is required
+bat_help() { 
+        "$1" --help | batcat 
+}
+alias bathelp='bat_help'
```

Comment:
I'm following something-heuristic convention

Reply:
can u link to the convention? will read up on it! even in single line function you need to appy semicolon inside function
``` bash
function foo { bar ; }
```
which wasn't present in the original thus gave error. 
Do you wish to have the bat_help function be just like how I mentioned above?

Reply:
LOL I mean't I'm followng my own fn naming convention for aliases with bat-help; git-graph

Reply:
Oh I'm pretty sure "-" denomination is not allowed for functions in POSIX shell that's why I renamed it from bat-help to bat_help. for alias it works yeah
I can change the alias bathelp to bat-help